.ci/docker/ci_commit_pins/pytorch.txt

```diff
@@ -1 +1 @@
-19eff28ff3f19b50da46f5a9ff5f4d4d213806fe
+288aa873831057b1eb7d747914ec4fdc76c23a80
```

Comment:
Should update this and the torch pin in a separate pr

Reply:
@dvorjackz but if I update the torch pin without other changes in the PR, the tests will fail due to the input to  torch._inductor.aoti_compile_and_package changed?

Reply:
True actually, let's use this instead though actually https://github.com/pytorch/executorch/pull/7273/, seems better for the sake of updating the pin correctly